examples/oauthbearer-default-flow.md

```diff
@@ -0,0 +1,80 @@
+Producer, Consumer and HighLevelProducer:
```

Comment:
Why is this just a markdown file instead of something executable?  And is it necessary to put the copyright header on all code snippets?